### PR TITLE
Added GitMixin

### PR DESCRIPTION
**Pull request by astiunov**
The first step from #108.

### CODE REVIEW

File: everware/spawner.py [hunk 1]
```diff
@@ -200,10 +272,10 @@ def build_image(self):
                 return image_name
 
         tmp_dir = mkdtemp(suffix='-everware')
-        self.git_executor = GitExecutor(self.form_repo_url, tmp_dir)
+        self.parse_url(self.form_repo_url, tmp_dir)
         self._add_to_log('Cloning repository %s' % self.repo_url)
         self.log.info('Cloning repo %s' % self.repo_url)
-        yield self.git_executor.exec()
+        yield self.prepare_local_repo()
```
Comment (betatim) [inline everware/spawner.py:278]: yeah, I like the name! 😎

**Comment by betatim** [comment]
Has someone else had time to look at this in detail? I haven't and won't for the next week. If we trust in travis then we could merge it. Thoughts?

**Comment by anaderi** [comment]
Thanks!

**Comment by betatim** [comment]
Closed? Why not merge? doubleconfused.com

**Comment by astiunov** [comment]
I added `stop` method to spawner. I don't know exactly if it helps, but as far as I'm concerned the problem with travis is in `stop` method - it throws exception, because dockerspawner tries to stop already destroyed container

**Comment by astiunov** [comment]
Ok, I think I'll add this with a new PR, so let wait a few

**Comment by astiunov** [comment]
But this can be merged anyway

**Pull request merged**